feat(pxe-with-full-private-dedicated): update dhcp conf (#8908)
ovh/docs: pages/bare_metal_cloud/dedicated_servers/pxe-with-full-private-dedicated/guide.en-gb.md

- Enter a primary configuration file (as an example, see file below).

```bash
{
"Dhcp4": {
    "interfaces-config": {
        "interfaces": [ "eth1" ]
    },

    "control-socket": {
        "socket-type": "unix",
        "socket-name": "kea4-ctrl-socket"
    },

    "lease-database": {
        "type": "memfile",
        "lfc-interval": 3600
    },

    "expired-leases-processing": {
        "reclaim-timer-wait-time": 10,
        "flush-reclaimed-timer-wait-time": 25,
        "hold-reclaimed-time": 3600,
        "max-reclaim-leases": 100,
        "max-reclaim-time": 250,
        "unwarned-reclaim-cycles": 5
    },

    "renew-timer": 900,
    "rebind-timer": 1800,
    "valid-lifetime": 3600,

    "client-classes": [
        {

>             "name": "iPXE",

Reviewer: ```suggestion
            "name": "iPXE script",
```
That would describe the class better IMO.

>             "test": "option[user-class].exists and substring(option[user-class].hex,0,4) == 'iPXE' and option[client-system].hex == 0x0007", # Détermine le type d'architecture, ici 64bits

Reviewer 2: ```suggestion
            "test": "option[user-class].exists and substring(option[user-class].hex,0,4) == 'iPXE' and option[client-system].hex == 0x0007", # Determines the type of architecture, in this case 64 bits
```

Author: @Y0Coss Edited. Sorry for these omissions.

            "next-server": "192.168.1.1", # Détermine l'IP de votre serveur tftp

>             "boot-file-name": "refind.pxe", # Détermine le script appelé par le binaire

Reviewer: ```suggestion
            "boot-file-name": "refind.pxe", # Determines the script loaded by the iPXE binary
```
All comments should be translated into English. I only touched this one because I'd like to change the wording.

        },
        {
            "name": "PXE UEFI",

>             "test": "not option[user-class].exists and option[client-system].hex == 0x0007", # Détermine le type d'architecture, ici 64bits

Reviewer 2: ```suggestion
            "test": "not option[user-class].exists and option[client-system].hex == 0x0007", # Determines the type of architecture, in this case 64 bits
```

```bash

> #!ipxe 

Reviewer: ```suggestion
#!ipxe
```
The trailing whitespace might break iPXE

> #

Reviewer: These changes do not seem related to the rest of the PR.

Author: > These changes do not seem related to the rest of the PR.

Agree. The file in the ressource zip file has changed so I have passed on the changes here but we could made a new PR. Can you tell me your preferences ?

Reviewer: @nalscher OK I get it now! However the ZIP hasn't changed since it was added in 0348bfd7cdd110bdd7ba3ab926dd19e2f8cf6205. It was likely a mistake that it contained these sleep calls and commented out lines. Can you please do the opposite and update the ZIP with the contents of this script (before your changes)?

Author: @sbraz Done. Tested on my servers, it works.
